### ISSUE TITLE
Fix test_that_checks_amo_logo_text_layout_and_title test

### ISSUE THREAD

**Issue by stephendonner** [mozilla/Addon-Tests#832]
After the fix for the ALT text in https://github.com/mozilla/Addon-Tests/pull/831 , test_that_checks_amo_logo_text_layout_and_title is failing due to the following image-path changes:

`13:50:49 ________ TestAmoLayout.test_that_checks_amo_logo_text_layout_and_title _________
13:50:49 [gw10] linux2 -- Python 2.7.9 /var/lib/jenkins/jobs/amo.dev.saucelabs/workspace/.venv/bin/python2.7
13:50:49 tests/desktop/test_layout.py:27: in test_that_checks_amo_logo_text_layout_and_title
13:50:49     assert '/img/app-icons/med/firefox.png' in home_page.amo_logo_image_source
13:50:49 E   assert '/img/app-icons/med/firefox.png' in 'https://addons-dev-cdn.allizom.org/static/img/icons/firefox.png?b=a9b9c79-570c07bb'
13:50:49 E    +  where 'https://addons-dev-cdn.allizom.org/static/img/icons/firefox.png?b=a9b9c79-570c07bb' = <pages.desktop.home.Home object at 0x7f090af6d710>.amo_logo_image_source`

**Comment by davehunt** [comment]
Fixed by #833
